LeaderboardBackend/Controllers/RunsController.cs

```diff
@@ -34,30 +38,32 @@ public async Task<ActionResult<RunViewModel>> GetRun(Guid id)
     }
 
     [Authorize]
-    [HttpPost("runs/create")]
-    [SwaggerOperation("Creates a new Run.", OperationId = "createRun")]
+    [HttpPost("/category/{id:long}/runs/create")]
+    [SwaggerOperation("Creates a new Run for a Category with ID `id`. This request is restricted to confirmed Users and Administrators.", OperationId = "createRun")]
     [SwaggerResponse(201)]
-    [SwaggerResponse(401)]
-    [SwaggerResponse(403)]
+    [SwaggerResponse(401, "The client is not logged in.", typeof(ProblemDetails))]
+    [SwaggerResponse(403, "The requesting User is unauthorized to create Runs.", typeof(ProblemDetails))]
+    [SwaggerResponse(404, "The Category with ID `id` could not be found.", typeof(ProblemDetails))]
     [SwaggerResponse(422, Type = typeof(ValidationProblemDetails))]
-    public async Task<ActionResult> CreateRun([FromBody] CreateRunRequest request)
+    public async Task<ActionResult<RunViewModel>> CreateRun([FromRoute] long id, [FromBody, SwaggerRequestBody(Required = true)] JsonDocument request)
```

Comment:
Surprised that this worked. Took a stab at setting `request` to be `JsonDocument` just to comply with `JsonSerializer.Deserialize`'s signature